js/main.js

```diff
@@ -1 +1,91 @@
-'use strict'
\ No newline at end of file
+'use strict'
+  let index = 0;
+  const body = document.querySelector('body');
+  const startBtn = document.getElementById('startBtn');
+  const headMessage = document.getElementById('headMessage');
+  const message = document.getElementById('message');
+  const category = document.getElementById('category');
+  const difficulty = document.getElementById('difficulty');
+  const btnArea = document.getElementById('clickArea');
+  const selectAnswer = [];
+      startBtn.addEventListener('click', () => {
+        message.innerHTML = "少々お待ち下さい";
+        fetch('https://opentdb.com/api.php?amount=10').then(response => {
+          return response.json()
+          .then(jsonDate => {
+          const question = jsonDate.results;
+          const kuizuPage = new Kuizu(question);
+          kuizuPage.createPage(index);
+        });
+      });  
+    })
+    class Kuizu {     
```

Comment:
好みの問題かもしれないですが…個人的には英語表現が難しい場合を除き
ローマ字での命名は避けたほうがいいと思います。
ちょっと長くなるとたちまち読みにくくなるので…。

```suggestion
    class Quiz {     
```

Reply:
以前もアドバイス頂いた部分だったと思いますが、完全に抜け落ちてしまっていました。癖づくように意識して行っていきます。アドバイスありがとうございます。